File: RangeSeekBar/src/main/java/com/jaygoo/widget/OnRangeChangedListener.java
package com.jaygoo.widget;

/**
 * ================================================
 * 作    者：JayGoo
 * 版    本：
 * 创建日期：2018/5/8
 * 描    述:
 * ================================================
 */
public interface OnRangeChangedListener {
    void onRangeChanged(DefRangeSeekBar view, float leftValue, float rightValue, boolean isFromUser);

    void onStartTrackingTouch(DefRangeSeekBar view, boolean isLeft);

    void onStopTrackingTouch(DefRangeSeekBar view, boolean isLeft);
}

File: android/libapp/src/main/java/com/jaygoo/widget/OnRangeChangedListener.kt
package com.jaygoo.widget

/**
 * ================================================
 * 作    者：JayGoo
 * 版    本：
 * 创建日期：2018/5/8
 * 描    述:
 * ================================================
 */
interface OnRangeChangedListener {
    fun onRangeChanged(view: DefRangeSeekBar, leftValue: Float, rightValue: Float, isFromUser: Boolean)

    fun onStartTrackingTouch(view: DefRangeSeekBar, isLeft: Boolean)

    fun onStopTrackingTouch(view: DefRangeSeekBar, isLeft: Boolean)
}